autoload/airline/extensions/tabline/buffers.vim

```diff
@@ -3,7 +3,14 @@
 
 scriptencoding utf-8
 
-let s:spc = g:airline_symbols.space
+let s:spc = get(g:, 'airline#extensions#tabline#pad_with_spaces', 1)
+      \ ? g:airline_symbols.space
+      \ : ''
+
+let s:ellipsis = get(g:, 'airline#extensions#tabline#ellipsis', '...')
+
+let s:buffers_label = get(g:, 'airline#extensions#tabline#buffers_label', 'buffers')
+let s:show_tab_type = get(g:, 'airline#extensions#tabline#show_tab_type', 1)
```

Comment:
please do not create more of those `s:` variables. That means, changing the configuration later, won't change the bufferline configuration. Instead move those initializiations at the place where they are actually used, e.g. `if get(g:, 'airline#extensions#tabline#show_tab_type', 1)`...